docs/contracts/inheritance.rst

```diff
@@ -392,33 +392,39 @@ and all functions that are reachable from there through function calls.
 It does not include the constructor code or internal functions that are
 only called from the constructor.
 
-Constructor functions can be either ``public`` or ``internal``. If there is no
+If there is no
 constructor, the contract will assume the default constructor, which is
-equivalent to ``constructor() public {}``. For example:
+equivalent to ``constructor() {}``. For example:
```

Comment:
Nitpick: perhaps you want to reformat these 3 lines into 2.

Reply:
Wouldn't it be much harder to review then?